cgi/product_multilingual.pl

```diff
@@ -453,7 +453,7 @@
 				$value = $value / 100 * $Nutriments{$nid}{dv} ;
 				$unit = $Nutriments{$nid}{unit};
 			}
-			$product_ref->{nutriments}{$nid} = unit_to_g($value, $unit);
+			$product_ref->{nutriments}{$nid} = unit_to_mmoll(unit_to_g($value, $unit), $unit);
```

Comment:
It looks a bit weird to chain the conversions even though at least one of them will actually do nothing.
I'm thinking it may be easier to understand if we do something like if ($nid eq "water-hardness") -> use unit_to_mmol, else unit_to_g